src/Microsoft.IdentityModel.Tokens/JsonWebKey.cs

```diff
@@ -155,21 +138,7 @@ private void Copy(JsonWebKey key)
         /// Gets or sets the 'key_ops' (Key Operations)..
         /// </summary>
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore, NullValueHandling = NullValueHandling.Ignore, PropertyName = JsonWebKeyParameterNames.KeyOps, Required = Required.Default)]
-        public IList<string> KeyOps
-        {
-            get
-            {
-                return _keyops;
-            }
-            set
-            {
-                if (value == null)
-                    throw LogHelper.LogException<ArgumentNullException>(LogMessages.IDX10001, "KeyOps");
-
-                foreach (string keyOp in value)
-                    _keyops.Add(keyOp);
-            }
-        }
+        public IList<string> KeyOps { get; private set; } = new List<string>();
```

Comment:
There is a change of behavior here, just want to make sure its intentional. Earlier, setting KeyOps would add values to existing array (always additive), now it replaces the whole list. The new behavior seems right to me, always additive does not make sense.